src/views/catalog/customize/CustomizeVirtualMachine.tsx

```diff
@@ -32,7 +33,15 @@ const CustomizeVirtualMachine: React.FC = () => {
     name,
     namespace: templateNamespace,
   });
-
+  const {
+    availableTemplatesUID,
+    loaded: loadedSources,
+    bootSourcesLoaded,
+  } = useTemplatesWithAvailableSource({
+    onlyAvailable: false,
+    onlyDefault: false,
+  });
+  const isBootSourceAvailable = availableTemplatesUID.has(template?.metadata?.uid);
```

Comment:
This will fetch a lot of unnecessary templates. 
You can just extract the function [here](https://github.com/kubevirt-ui/kubevirt-plugin/blob/main/src/views/catalog/templatescatalog/hooks/useTemplatesWithAvailableSource.ts#L38-L61) and use it also in this component

Source: https://github.com/kubevirt-ui/kubevirt-plugin/blob/main/src/views/catalog/templatescatalog/hooks/useTemplatesWithAvailableSource.ts